base/constants.go

```diff
@@ -5,4 +5,8 @@ const (
 	// The username of the special "GUEST" user
 	GuestUsername = "GUEST"
 	ISO8601Format = "2006-01-02T15:04:05.000Z07:00"
+
+	ViewQueryPageSize = 5000  // This must be greater than 1, or the code won't work due to windowing method
```

Comment:
It seems like a sensible value to me, but would it be much work to make it configurable? Perhaps as an optional arg in the POST to _repair? Just trying to think of things that might catch us out in different envs...

Reply:
Yep, great idea!  I defaulted it to 5000 but it can now be overridden via:

```
{
  "dry_run": true,
  "view_query_page_size": 500,
  "repair_jobs": [
     {
     	"type": "RepairRevTreeCycles"
     }	
  ]
}
```